control/templates/shared/add_vhost_test.html

```diff
@@ -54,7 +54,7 @@ <h2>Test custom domain</h2>
     <input type="hidden" name="confirm" value="on">
     {% if valid[domain] == (true, true) or valid[prefixed] == (true, true) %}
         <input type="submit" class="btn btn-outline-primary" value="Looks good, add domain">
-    {% else %}
+    {% elif valid[domain] != (none, none) or valid[domain] != (none, none) %}
```

Comment:
Do you really mean for both sides of the `or` to be the same, or should one be `valid[prefixed]` ?